app/models/user/reputation_tokens/code_contribution_token.rb

```diff
@@ -1,18 +1,18 @@
 class User::ReputationTokens::CodeContributionToken < User::ReputationToken
-  params :repo, :pr_id
+  params :repo, :pr_node_id, :pr_number
```

Comment:
As said before, I've kept the `pr_` prefix here as this code does not have a clear relation to a pull request, which could make `node_id` and `number` confusing.